mfes/content/src/components/CommonCollapse.tsx

```
@@ -179,12 +146,98 @@ export const CommonCollapse: React.FC<CommonAccordionProps> = ({
+    return result;
+  };
+  const fetchDataTrack = async () => {
+    try {
+      //@ts-ignore
+      if (TrackData && item?.children) {
+        for (let i = 0; i < TrackData.length; i++) {
+          //@ts-ignore
+          if (TrackData[i]?.courseId) {
+            //merge offlien and online
+            //@ts-ignore
+            const mergedArray = [...TrackData[i]?.completed_list];
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)